licenses/tests/test_models.py

```diff
@@ -285,3 +289,31 @@ def test_superseded(self):
         lic2 = LicenseFactory(is_replaced_by=lic1)
         self.assertTrue(lic2.superseded)
         self.assertFalse(lic1.superseded)
+
+
+class TranslationBranchModelTest(TestCase):
+    def test_stats(self):
```

Comment:
Can we add a test for when a branch is newly created and there are no translated or untranslated entries?

Reply:
Is that scenario possible? Any new translation branch will be based off 'develop', which already has messages.

Maybe if they were adding a brand new language... which we currently won't notice, since we only check the status of known languages in Transifex... maybe we should modify the logic to look for brand new languages?  Then add this test, among others?  What do you think?